client/app/templates/project.hbs

```diff
@@ -18,6 +18,17 @@
         <strong>Primary Applicant:</strong> {{@model.dcpApplicantCustomerValue}}
       </p>
 
+      <p>
+        <strong>Project Brief:</strong> {{@model.dcpProjectbrief}}
+      </p>
+
+      <h2>Status</h2>
+      
+      {{#each (filter-by 'statuscode' 'In Progress' @model.milestones) as |milestone|}}
+        <strong class="label dark-gray">{{milestone.dcpMilestoneValue}}</strong>
+      {{/each}}
+      <p></p>
```

Comment:
delete this unecessary `<p></p>` here?

If a line break is needed, you can use `<br>`